nginx-pfe-redirects.conf

```diff
@@ -0,0 +1,75 @@
+set $proxy_path "www.zooniverse.org";
+
+# FEM project assets
```

Comment:
Does the ordering of this directive, above and separate from the rest of the `projects/` pieces (starting on L37), matter?

Reply:
Yeah, the ordering is important in that it's gotta be before the other projects matcher. And though I grouped it with the other asset matcher, it could be moved further down.

In this case, before matching with `~* ^/projects`, I want to check for FEM assets (the assets and _next folders) and return them first as they're exceptions to the exceptions. I could switch the latter to a prefix matcher since they have a lower priority than regex matches, but this way seems more obvious: match `/projects` plus those folders explicitly and route to FEM, or match just `^/projects` and point to FEM. 

The FEM project redirects file itself is included first for the same reason.